app/Http/Controllers/Service/Admin/VouchersController.php

```diff
@@ -38,6 +42,97 @@ public function create()
         return view('service.vouchers.create', compact('sponsors'));
     }
 
+    /**
+     * Show the void Vouchers form
+     *
+     * @return Factory|view
```

Comment:
Apparently, it can according to phpstorm's code reflection. :man_shrugging:
![Selection_999(051)](https://user-images.githubusercontent.com/25037036/76534895-fe5fe180-6471-11ea-9695-a8b18e4a3164.png)